libs/langgraph/langgraph/graph/state.py

```diff
@@ -21,6 +21,7 @@
 from langchain_core.runnables.base import RunnableLike
 from langchain_core.runnables.utils import create_model
 from pydantic import BaseModel
+from pydantic.v1 import BaseModel as BaseModelV1
```

Comment:
does this crash if the user has pydantic v1 installed instead?

Reply:
no, tested on 1.10.18